site/docs/BestPractice-ServingOverHTTP.md

```diff
@@ -44,7 +44,8 @@ A standard GraphQL POST request should use the `application/json` content type,
 ```js
 {
   "query": "...",
-  "variables": { variable1: value, ... }
+  "oprationName": "...",
```

Comment:
OMG, sorry for the typo :flushed:

I was also thinking about adding a note about optional fields. In this case we need to mention that `variables` is optional as well. WDYT?